### ISSUE TITLE
homeless-try: `try` in `comptime` blocks should be allowed

### ISSUE THREAD

**Issue by DonIsaac** [DonIsaac/zlint#177]
This code is valid. I wonder how many other edge cases there are like this.

```zig
fn foo() void {
  comptime {
    // failure -> compiler error
    const x = try bar();
    _ = x;
  }
}
```
